### PR TITLE
use clientwrapper instead of client

### PR DESCRIPTION
**Pull request by odinuv**
preparation for https://keboola.atlassian.net/browse/SOX-323
BC break, needs to be released as major

### CODE REVIEW

File: tests/ArtifactsTest.php [hunk 1]
```diff
@@ -856,15 +878,17 @@ private function downloadAndAssertStorageFile(
         array $tags,
         bool $unzip = true
     ): void {
-        $storageClient = $this->getStorageClient();
+        $storageClient = $this->getStorageClientWrapper();
```
Comment (MiroCillik) [inline tests/ArtifactsTest.php:881]: Tady bych prejmenoval $storageClient -> $clientWrapper

**Review by MiroCillik — APPROVED** [review]
LGTM

File: tests/ArtifactsTest.php (hunk 1 above)
Comment (odinuv) [reply tests/ArtifactsTest.php]: jasně

File: tests/FilesystemTest.php [hunk 2]
```diff
@@ -124,13 +124,22 @@ public function testArchiveAndExtractDir(): void
...
+            ),
+            sprintf(
+                '%s/archive-extract-test%stest%stest2.txt',
+                $temp->getTmpFolder(),
+                DIRECTORY_SEPARATOR,
+                DIRECTORY_SEPARATOR
+            ),
+        ];
+        sort($expectedFiles);
+        sort($files);
```
Comment (MiroCillik) [inline tests/FilesystemTest.php:141]: 👍

**Pull request merged**